### PR TITLE
Convert sites to HTTPS

### PR DESCRIPTION
**Pull request by luciusbono**
Changed sites that were listed as HTTP to HTTPS, to unblock https://github.com/mozilla/dinobuildr/pull/107. Removed two sites that appear to not support HTTPS

### CODE REVIEW

**Comment by luciusbono** [comment]
@potch: this PR would remove http://micropipes.com/temp/ because it does not support HTTPS
@wagnerand: this PR would remove "http://emanueladamiani.com/photon/welcome-june.jpg comment=\"Welcome new hires\" because it does not support HTTPS

Can you both review and weigh in?

**Comment by brassy-** [comment]
Sure, let's sinc and move it. We're going to have a new one for July!

**Comment by luciusbono** [comment]
@tristanthomas got the site owner of micropipes.com/temp/ to update their site to support HTTPS, so we're re-adding it!

File: state.json [hunk 1]
```diff
@@ -61,8 +61,8 @@
                 "name": "whimsy",
                 "commands": [
                     "content type=html comment=\"Whimsycorn\" content=\"<style>@keyframes AnimationName{0%{background-position:center,100vw 50%}100%{background-position:center,0vw 50%}}body{background:url(https://raw.githubusercontent.com/bwinton/whimsy/f8c52e336233897ba37aa265e2fccdaa008a2ca1/wheeeeee.png)no-repeat center,linear-gradient(to right,red 0%,orange 17vw,yellow 33vw,#0f0 50vw,blue 67vw,violet 83vw,red 100vw);height:100%;background-size:90vh,cover;margin:0;animation:AnimationName 180s linear infinite;}</style><body>\"",
-                    "http://micropipes.com/temp/",
-                    "https://trends.google.com/trends/hottrends/visualize?nrow=3&ncol=3"
+                    "https://micropipes.com/temp/",
+		    "https://trends.google.com/trends/hottrends/visualize?nrow=3&ncol=3"
```
Comment (mythmon) [inline state.json]: You introduced some mixed tabs/spaces here. Can you fix that?

Comment (luciusbono) [reply state.json]: Gross, tabs! fixed in https://github.com/mozilla/moz-corsica/pull/123/commits/b4084a517208d9d51c90303ba352a3c886a34850

**Pull request merged**